app/validators/wpcc/salary_threshold_validator.rb

```diff
@@ -0,0 +1,22 @@
+module Wpcc
+  class SalaryThresholdValidator < ActiveModel::Validator
+    SALARY_THRESHOLDS  = [5876.00, 489.67, 452.00, 113.00].freeze
+    SALARY_FREQUENCIES = %w[year month fourweeks week].freeze
+    LOW_SALARY_THRESHOLDS = SALARY_FREQUENCIES.zip(SALARY_THRESHOLDS).to_h
```

Comment:
Also wondering whether these could/should be in a `yaml` file? 
I know the frequencies are set in the form already, perhaps not the thresholds. Would it be better to have the frequencies set in one place only? @margOnline @tomas-stefano I'll ask your opinions on this as well as I am not sure of the best solution. this one may be fine!